### PR TITLE
Issue 64 component blog

### PR DESCRIPTION
**Pull request by ChipsVII**

### CODE REVIEW

File: src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5 [hunk 1]
```diff
@@ -18,4 +22,22 @@
     </div>
   <?php endif; ?>
 
+
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
```
Comment (LupusVII) [inline src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5:26]: Remove this line. FA will be inserted elsewhere

File: src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5 [hunk 2]
```diff
@@ -18,4 +22,22 @@
...
+<link rel="stylesheet" type="text/css" href="assets/jssocials/css/jssocials.css" />
+<link rel="stylesheet" type="text/css" href="assets/jssocials/css/jssocials-theme-flat.css" />
+<script
+        src="https://code.jquery.com/jquery-3.6.0.min.js"
+        integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4="
+        crossorigin="anonymous"></script>
+ <script src="assets/jssocials/js/jssocials.min.js"></script>
+ <script src="assets/jssocials/js/jssocials.override.fav5.js"></script>
+<script>
+  $("#share").jsSocials({
```
Comment (LupusVII) [inline src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5:36]: Remove JSSocials dependency and generate manually the share links

File: src/Classes/Backend/Block.php [hunk 3]
```diff
@@ -299,6 +283,31 @@ public function getMode(): string
         return $mode;
     }
 
+    protected function checkRequierments(): bool
```
[the remark this answers is not in the document]
Comment (ChipsVII) [reply src/Classes/Backend/Block.php]: Done

File: src/EventListener/ReplaceInsertTagsListener.php [hunk 4]
```diff
@@ -56,6 +57,7 @@ public function onReplaceInsertTags(
         $elements = explode('::', $insertTag);
         $key = strtolower($elements[0]);
         if ('sg' === $key) {
```
[the remark this answers is not in the document]
Comment (ChipsVII) [reply src/EventListener/ReplaceInsertTagsListener.php:58]: done

File: src/EventListener/LoadDataContainerListener.php [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        $this->coreConfigurationManager = $coreConfigurationManager;
+    }
+
+    public function __invoke(string $table): void
+    {
+        try {
+            /** @var CoreConfig */
+            $config = $this->coreConfigurationManager->load();
+            switch ($table) {
+                case 'tl_news':
```
[the remark this answers is not in the document]
Comment (ChipsVII) [reply src/EventListener/LoadDataContainerListener.php]: Done!

File: src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5 [hunk 6]
```diff
@@ -4,9 +4,13 @@
...
   <?= $this->articles ?>
 
-  <!-- indexer::stop -->
-  <p class="back"><a href="<?= $this->referer ?>" title="<?= $this->back ?>" class="btn"><?= $this->back ?></a></p>
-  <!-- indexer::continue -->
+  <div>
+    <!-- indexer::stop -->
+    <p class="back fl-left"><a href="<?= $this->referer ?>" title="<?= $this->back ?>" class="btn"><?= $this->back ?></a></p>
+    <!-- indexer::continue -->
+    <div id="share" class="fl-right"></div>
```
[the remark this answers is not in the document]
Comment (ChipsVII) [reply src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5]: done

File: src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5 (hunk 1 above)
Comment (ChipsVII) [reply src/Resources/public/contao_files/templates/smartgear/mod_newsreader.html5]: done

**Pull request merged**